Makefile

```diff
@@ -165,23 +167,37 @@ prepare-all-in-one-deploy-file: ## Prepares all in one deploy file
 .PHONY: e2e
 CURRENT_DIRECTORY := $(shell pwd)
 JENKINS_API_HOSTNAME := $(shell $(JENKINS_API_HOSTNAME_COMMAND))
-e2e: docker-build ## Runs e2e tests, you can use EXTRA_ARGS
+USE_ORGANIZATION := false
+e2e: cr-build ## Runs e2e tests, you can use EXTRA_ARGS
 	@echo "+ $@"
 	@echo "Docker image: $(DOCKER_REGISTRY):$(GITCOMMIT)"
+ifeq ($(KUBERNETES_PROVIDER),minikube)
```

Comment:
KUBERNETES_PROVIDER has not been instantiated ? How will this be checked ? Through envVars ?